apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/lib/site-plan/use-site-plan.js

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from '@wordpress/element';
+import { useCallback } from 'react';
+import request from '../../../../../../packages/wpcom-proxy-request/src/index';
```

Comment:
I'm pretty sure there must be a better way to import this lib!

Reply:
something like this? https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/wp/browser.js#L3

Reply:
🤦‍♂️